### PR TITLE
docs: Fix typo in app/building-your-application/caching

### PR DESCRIPTION
**Pull request by hiddenest**
- fix typo in [Building Your Application: Caching](https://nextjs.org/docs/app/building-your-application/caching#fetch-optionsnexttag-and-revalidatetag)
- `options.next.tags` is correct (current: `options.next.tag`)

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 14f1c36972d99a20089f1c1566f68ae8ee7e5b68

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/01-building-your-application/04-caching/index.mdx
Comment (leerob) [inline docs/02-app/01-building-your-application/04-caching/index.mdx:411]: Could we run prettier here please?

**Review by leerob — DISMISSED** [review]

File: docs/02-app/01-building-your-application/04-caching/index.mdx
Comment (leerob) [reply docs/02-app/01-building-your-application/04-caching/index.mdx:411]: Wait, I can't tell from the diff if it's needed - approving and if it passes, it will merge!

Comment (hiddenest) [reply docs/02-app/01-building-your-application/04-caching/index.mdx:411]: ran prettier and pushed it :)

**Review by leerob — APPROVED** [review]

**Pull request merged**
